main.js

```diff
@@ -0,0 +1,94 @@
+// 1. Вивести в console числа, від 10 до 25 з кроком 0.5 (10 й 25 включно)
+for (let i = 10; i <= 25; i = i + 0.5) {
+    console.log(i);
+}
+
+// 2. Запитати та зберегти число введене із prompt.
+const USER_NUM = +prompt("Enter a positive number: ");
+console.log(USER_NUM);
+
+// 3. Визначати чи це число просте
+let isPrime = true;
+if (USER_NUM === 1) {
+  console.log("1 is neither prime nor composite number."); 
+} else if (USER_NUM > 1) {
+  for (let i = 2; i < USER_NUM; i++) {
+    if (USER_NUM % i === 0) {
+      isPrime = false; 
+      console.log(`${USER_NUM} is not a prime number.`);
+      break; 
+    } if (isPrime) {
+      console.log(`${USER_NUM} is a prime number.`);
+      break;
+    }
+  }
+}
+   
+
+// 4. Запитати та зберегти суму покупки введене із prompt. Обрахувати знижку та повідомити користувачу значення знижки. Якщо сума покупки до 100 UAН - знижка 3%; якщо сума покупки до 200 UAH - знижка 5%; якщо сума покупки перевищує 200 UAH - знижка 7%
+const PURCHASE_AMOUNT = +prompt("How much money did you spent?");
+console.log(PURCHASE_AMOUNT);
+
+if (PURCHASE_AMOUNT < 100 && PURCHASE_AMOUNT > 0) {
+    const DISCOUNT_3 = (PURCHASE_AMOUNT * 0.03).toFixed(2);
+    console.log(`Your discount is ${DISCOUNT_3} - 3%`);
+} else if (PURCHASE_AMOUNT >= 100 && PURCHASE_AMOUNT < 200) {
+    const DISCOUNT_5 = (PURCHASE_AMOUNT * 0.05).toFixed(2);
+    console.log(`Your discount is ${DISCOUNT_5} - 5%`);
+} else if (PURCHASE_AMOUNT >= 200) {
+    const DISCOUNT_7 = (PURCHASE_AMOUNT * 0.07).toFixed(2);
+    console.log(`Your discount is ${DISCOUNT_7} - 7%`);
+}
+
```

Comment:
Shorter and more convenient code

```suggestion
let discount = null;
if (PURCHASE_AMOUNT < 100 && PURCHASE_AMOUNT > 0) {
    discount = (PURCHASE_AMOUNT * 0.03).toFixed(2);
} else if (PURCHASE_AMOUNT >= 100 && PURCHASE_AMOUNT < 200) {
    discount = (PURCHASE_AMOUNT * 0.05).toFixed(2);
} else if (PURCHASE_AMOUNT >= 200) {
    discount = (PURCHASE_AMOUNT * 0.07).toFixed(2);
}

console.log(`Your discount is ${discount}%`);
```